spec/support/acceptance/adds_shoulda_matchers_to_project.rb

```diff
@@ -118,6 +119,10 @@ def integrates_with_rspec?(test_framework)
       test_framework == :rspec
     end
 
+    def integrates_with_rspec_expectations?(test_framework)
```

Comment:
This would be `integrates_with_cucumber?`. You'd have to modify `test_helper_files_for` to do:

``` ruby
if integrates_with_cucumber?(test_framework)
  files << 'features/support/env.rb'
end
```